sentry-core/src/main/java/io/sentry/core/transport/AsyncConnection.java

```diff
@@ -15,11 +15,13 @@
 import java.util.concurrent.RejectedExecutionHandler;
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeUnit;
+import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.jetbrains.annotations.TestOnly;
 
 /** A connection to Sentry that sends the events asynchronously. */
+@ApiStatus.Internal // TODO: check if its only internal
```

Comment:
@bruno-garcia can it be used by DI as well?

Reply:
The Hub ctor takes SentryOptions so for now lets keep the rest of the API internal so we can move things around a bit.